### PR TITLE
Add Archi currency: RUB & TRY

### PR DESCRIPTION
**Pull request by XinxingChen**
Update Archi & Cat
Update Archi live gif with specific emotions

### CODE REVIEW

File: Brain/ArchiBoT/Custom/archi.siml [hunk 1]
```diff
@@ -297,6 +320,68 @@
...
+		
+		<Model>
+			<Pattern>Archi Break</Pattern>
+			<Pattern>Archi Knock</Pattern>
+			<Response>https://i.loli.net/2019/03/12/5c8770790ef23.gif</Response>
+		</Model>
+		
+		<Model>
+			<Pattern>Archi Firlt</Pattern>
+			<Pattern>Archi Firlting</Pattern>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/archi.siml]: I assume you mean "flirt" and "flirting" :p

File: Brain/ArchiBoT/Custom/archi.siml [hunk 2]
```diff
@@ -297,6 +320,68 @@
...
+		<Model>
+			<Pattern>Archi Angry</Pattern>
+			<Pattern>Archi Bite</Pattern>
+			<Pattern>Archi Roar</Pattern>
+			<Response>
+				<Random>
+					<Item>https://i.loli.net/2019/03/12/5c8770816d7fd.gif</Item>
+					<Item>https://i.loli.net/2019/03/12/5c87708252c9d.gif</Item>
+					<Item></Item>
+					<Item></Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/archi.siml]: Those 2 entries should be removed

File: Brain/ArchiBoT/Custom/archi.siml (hunk 1 above)
Comment (XinxingChen) [reply Brain/ArchiBoT/Custom/archi.siml]: What a pity that GitHub does not have auto correct function!

File: Brain/ArchiBoT/Custom/archi.siml (hunk 2 above)
Comment (XinxingChen) [reply Brain/ArchiBoT/Custom/archi.siml]: My bad memory >.<

**Pull request merged**